src/frontend/src/pages/Groups/GroupsDataTable.tsx

```diff
@@ -0,0 +1,48 @@
+// @ts-nocheck
+
+// Most carbon components lack typescript definitions
+// this is notable around something complex like the DataTable used here,
+// so we disable ts for the whole file as a workaround this is exported back to to View though, which still enables TS.
+
+// see here for the current progress of adding TypeScript support to @carbon/react
+// https://github.com/carbon-design-system/carbon/discussions/10752#discussioncomment-4710978
```

Comment:
this hack is to get around the (lack of) typescript support for Carbon React components

See here, https://github.com/carbon-design-system/carbon/discussions/10752#discussioncomment-4710978